plugin-management-library/pom.xml

```diff
@@ -27,7 +27,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.9</version>
```

Comment:
not sure about this one, cc @oleg-nenashev should this come from the jenkins bom?